requirements.txt

```diff
@@ -0,0 +1,4 @@
+autogluon[tabular]
```

Comment:
autogluon.tabular[all]

Reply:
Indeed, sorry I missed your comment. I was not able to get it working so I skip it, will close and reopen if I get the time.

Reply:
probably we need to wait until AG 0.8, since latest release is missing some key logic that we need for this repo